core/templates/services/playthrough.service.ts

```diff
@@ -274,19 +284,28 @@ export class PlaythroughService {
    * If none of the issue types have been discovered, returns null instead.
    */
   private createNewPlaythrough(): Playthrough | null {
-    if (this.misTracker && this.misTracker.foundAnIssue()) {
+    if (
+      this.misTracker &&
```

Comment:
@JeeveshGarg just one further request -- please read through your PR's Files Changed tab, and remove all changes that aren't necessary, so that the diff that you're submitting is the minimum that is required. In some cases you seem to be making modifications to things that don't need changing.

Thanks!

Reply:
Done.